Automattic/jetpack
# Social: Fix Mastodon Fediverse creator og fatals

Fixes fatals introduced in https://github.com/Automattic/jetpack/pull/38198

## Proposed changes:

*

### Other information:


## Jetpack product discussion


p1722326166790739-slack-C034JEXD1RD
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Sandbox this change to WP.com 
* Enable Jetpack Social, and connect your site to a test Mastodon account.
* Go to Posts > Add New
* Write a post and publish it. In the post sidebar, ensure the post is shared to Mastodon.
* Open the post. Make sure you don't get any fatals in the logs. Without this change fatals should be present
